src/execution_plan/ops/op_node_by_id_seek.c

```diff
@@ -0,0 +1,97 @@
+/*
+* Copyright 2018-2019 Redis Labs Ltd. and Contributors
+*
+* This file is available under the Redis Labs Source Available License Agreement
+*/
+
+#include "op_node_by_id_seek.h"
+
+#define MIN(a,b) (((a) < (b)) ? (a) : (b))
+#define MAX(a,b) (((a) < (b)) ? (b) : (a))
+
+// Checks to see if operation index is within its bounds.
+static inline bool _outOfBounds(OpNodeByIdSeek *op) {
+    /* Because currentId starts at minimum and only increases
+     * we only care about top bound. */
+    if(op->currentId > op->maxId) return true;
+    if(op->currentId == op->maxId && !op->maxExclusive) return true;
+    return false;
+}
+
+OpBase* NewOpNodeByIdSeekOp
+(
+    const AST *ast,
+    unsigned int nodeRecIdx,
+    NodeID minId,
+    NodeID maxId,
+    bool minExclusive,
+    bool maxExclusive
```

Comment:
`minExclusive` and `maxExclusive` are misnamed - should be `Inclusive`.

Reply:
renamed.